src/SourceBuild/content/Directory.Build.props

```diff
@@ -99,6 +99,11 @@
 
     <!-- TargetFrameworkDefaults.props -->
     <NetCurrent>net9.0</NetCurrent>
+
+    <!-- Set up the build phase since the orchestrator switch is passed.
+         From RepoDefaults.props. -->
+    <DotNetBuild>true</DotNetBuild>
+    <DotNetBuildPhase>Orchestrator</DotNetBuildPhase>
```

Comment:
Did you intentionally place this into the property group that is only evaluated when not using the Arcade Sdk?

Reply:
Yes, because when the arcade SDK is in use, these are automatically set based on the DotNetBuildInnerRepo, DotNetBuildRepo, and DotNetBuildOrchestrator values.